src/util/configHelpers.js

```diff
@@ -1,3 +1,4 @@
+import defaultConfig from '../config/configDefault';
```

Comment:
This is wrong import ("defaultConfig**s**" is passed as a parameter, but the file itself should not be imported).